taxi/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
```

Comment:
fix the imports. python modules first, then installed modules, then your own and each group should be separated by blank line

2 blank lines after the last import